lib/fastboot-info.js

```diff
@@ -30,6 +31,20 @@ FastBootInfo.prototype.extractCookies = function(request) {
   return {};
 };
 
+FastBootInfo.prototype.host = function() {
+  if (!this.hostWhitelist) {
+    throw new Error('You must provide a hostWhitelist to retrieve the host');
+  }
+
+  var host = this.request.headers.host;
+
+  if (this.hostWhitelist.indexOf(host) === -1) {
```

Comment:
This should probably support regex or wildcards, so you can say `"*.example.com"` instead of enumerating all of the possible combinations.